certbot/compat/os.py

```diff
@@ -0,0 +1,20 @@
+"""
+This compat modules extends native capabilities of core os module to handle correctly platform
+specific operations (eg. chown, chmod, geuid).
+This module is intended to replace standard os module throughout certbot projects (except acme).
+"""
+from __future__ import absolute_import
+
+# Expose everything from standard os package to make current package a complete replacement of os.
+from os import *  # type: ignore  # pylint: disable=wildcard-import,unused-wildcard-import,redefined-builtin,os-module-forbidden
+
+# Monkey patch ourselves to get os attributes that are not in __all__ (so not from os import *).
+import os as std_os  # pylint: disable=os-module-forbidden
+import sys as std_sys
+ourselves = std_sys.modules[__name__]
```

Comment:
nit: This will almost certainly not be a problem, but can we pick a slightly more obscure name that we don't think could be used (internally) in the `os` module in the future. What about `certbot_os`?